x-pack/legacy/plugins/ingest/server/libs/policy.ts

```diff
@@ -258,11 +268,21 @@ export class PolicyLib {
       updated_on: new Date().toString(),
       updated_by: (user as FrameworkAuthenticatedUser).username || 'Fleet (system action)',
     });
-    // TODO fire events for fleet that update was made
 
+    await this._triggerPolicyUpdatedEvent(user, 'updated', id);
     // TODO create audit/history log
     // const newPolicy = await this.adapter.create(policyData);
 
     return policy as Policy;
   }
+
+  private async _triggerPolicyUpdatedEvent(
```

Comment:
can this use the `PolicyUpdateHandler` typing?

Reply:
It makes no really sense to use `PolicyUpdateHandler` typings here it's just a private method that is going to call registered handlers